### DISCUSSION TITLE
Type error for getDefaultConfig

### DISCUSSION THREAD

**Discussion by petermazzocco** [family/connectkit#281] [Q&A] [answered]
I just launched a new dApp using connectkit and I am getting an odd TypeScript error when importing `getDefaultConfig`.

The following is the error:
```
Could not find a declaration file for module 'connectkit'. '/Users/name/Desktop/VSCode/project/node_modules/connectkit/build/index.es.js' implicitly has an 'any' type.
```

This hasn't happened before so I am curious if there's a type error in the module or if I just need to reinstall my node modules 

I am running the following versions:

```
"connectkit": "^1.5.0",
"viem": "^1.5.0",
"wagmi": "^1.3.9"
```

**Comment by lochie** [comment, marked as answer]
We're currently looking into this, but a temporary fix is setting `"resolvePackageJsonExports": false` in your `tsconfig.json` file if you need something to keep you moving.

**Reply by petermazzocco** [reply]
Thanks for the quick reply. I'll make note of it and change it over!

**Reply by petermazzocco** [reply]
When I implement the changes, the error goes away but I am hit wit a "cant resolve fs" error now when importing any component like the ConnectKitButton

**Reply by lochie** [reply]
That one is something to do with WalletConnect's SDK, a solution can be found here: https://github.com/wagmi-dev/wagmi/issues/2300#issuecomment-1541425648

**Comment by dylanngph** [comment]
Waiting for fix...
